package.json

```diff
@@ -44,7 +44,7 @@
     "seed": "tsx scripts/seed-local-env.ts",
     "start": "pnpm run local:setup",
     "test": "vitest",
-    "test:e2e": "CYPRESS_BASE_URL=$HIVE_APP_BASE_URL cypress run",
+    "test:e2e": "CYPRESS_BASE_URL=$HIVE_APP_BASE_URL cypress run --browser chrome",
```

Comment:
I noticed that the chrome is not used by default on CI. Both locally and on CI chrome seems to be less flaky.